### DISCUSSION TITLE
Returning to bootstrap, what's changed?

### DISCUSSION THREAD

**Discussion by garrett85** [twbs/bootstrap#36388] [Q&A]
So it's been a few years since I looked at bootstrap, last time I downloaded bootstrap there was pretty much a full blown template site available. It was full of pages, menus and buttons, the developer just had to add/remove/edit the provided content. Now that I've returned to bootstrap there's nothing here but a CSS directory and a JS directory. What happened, what did I miss? I've also noticed that the CSS and JS files are not laid out in any particular fashion, the whole document seems to be one giant string of text jumpled together. This isn't the bootstrap I remember.

**Comment by mdo** [comment]
> So it's been a few years since I looked at bootstrap, last time I downloaded bootstrap there was pretty much a full blown template site available.

Assuming you're downloading the distribution zip files and not the full release, our downloads have always only included compiled assets. This has been since v3, which was hasn't really been updated in a like 4-5 years. Downloads for v4 and v5 are nearly identical, but v5 does have a few more options with RTL.

Perhaps you're talking about the documentation? That's available for anyone who wants to clone this repository.

> It was full of pages, menus and buttons, the developer just had to add/remove/edit the provided content.

This is what our documentation exists for, however, you're more than welcome to [explore the growing examples page](https://getbootstrap.com/docs/5.2/examples/) of our site where we showcase full pages, component variations, and custom snippets for any project.

> What happened, what did I miss? I've also noticed that the CSS and JS files are not laid out in any particular fashion, the whole document seems to be one giant string of text jumpled together.

Not sure what you're referring to here—perhaps the minification of our files? We minify everything to reduce file size.

> This isn't the bootstrap I remember.

🤷🏻‍♂️ Not sure what to tell you :).
